pkg/agent/openflow/multicast_test.go

```diff
@@ -80,3 +86,61 @@ func Test_featureMulticast_initFlows(t *testing.T) {
 		})
 	}
 }
+
+// If any test case fails, please consider setting binding.MaxBucketsPerMessage to a smaller value.
+func TestMulticastReceiversGroupMaxBuckets(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	fs := &featureMulticast{
+		bridge: binding.NewOFBridge(bridgeName, ""),
+	}
+	fakeOfTable := ovsoftest.NewMockTable(ctrl)
+	MulticastOutputTable.ofTable = fakeOfTable
+	defer func() {
+		MulticastOutputTable.ofTable = nil
+	}()
```

Comment:
Is the reason that each sub-test runs concurrently and they share the same mock object, and there is a risk of data races and unexpected behavior?

Reply:
I forgot to reply, sorry
No the risk is not so much about concurrency issues, because by default subtests are not run in parallel
The risk is about expectations set in one sub-test leaking to the next sub-test(s), because the mock object is common